File: Programming Basics/JavaScript/Conditional Statements/Exercises/lunch-break.js
function solve(input) {
    const seriesName = input[0];
    const episodeDuration = Number(input[1]);
    const breakDuration = Number(input[2]);

    const requiredTime = episodeDuration + 0.125 * breakDuration + 0.25 * breakDuration;

    if (requiredTime <= breakDuration) console.log(`You have enough time to watch ${seriesName} and left with ${Math.ceil(breakDuration - requiredTime)} minutes free time.`);
    else console.log(`You don't have enough time to watch ${seriesName}, you need ${Math.ceil(requiredTime - breakDuration)} more minutes.`);
}

File: Programming Basics/C++/Conditional Statements/Exercises/LunchBreak/LunchBreak.cpp
#include <iostream>
#include <math.h>
#include <string>
using namespace std;

int main()
{
	// 1. Read input
	string seriesName;
	int episodeDuration, breakDuration;

	getline(cin, seriesName);
	cin >> episodeDuration >> breakDuration;

	// 2. Calculate required time
	double lunchDuration = 0.125 * breakDuration;
	double restDuration = 0.25 * breakDuration;
	double necessaryDuration = episodeDuration + lunchDuration + restDuration;

	// 3. Print output
	if (breakDuration >= necessaryDuration)
		cout << "You have enough time to watch " << seriesName << " and left with " << ceil(breakDuration - necessaryDuration) << " minutes free time." << endl;
	else
		cout << "You don't have enough time to watch " << seriesName << ", you need " << ceil(necessaryDuration - breakDuration) << " more minutes." << endl;
}
